tests/integration/components/helpers/trigger-tooltip-event-test.js

```diff
@@ -0,0 +1,21 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import { triggerTooltipEvent } from '../../../helpers/ember-tooltips';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('tooltip-on-element', 'Integration | Helpers | triggerTooltipEvent', {
+  integration: true,
+});
+
+test('triggerTooltipEvent throws appropriate error', function(assert) {
```

Comment:
Should we test for the events actually being emitted here? I have experienced the brittle nature of events in Ember test before so I'm ok if we don't do that - am just suggesting it.